addons/account/tests/common.py

```diff
@@ -830,18 +849,37 @@ def compare_taxes_computation_values(results, expected_values, rounding):
 
         params = test['params']
         is_round_globally = params['is_round_globally']
+        excluded_special_modes = params['excluded_special_modes']
         rounding = 0.000001 if is_round_globally else 0.01
         compare_taxes_computation_values(results['results'], test['expected_values'], rounding)
 
-        # Check the reverse in case of round_globally.
+        # Check the special modes in case of round_globally.
         if is_round_globally:
-            compare_taxes_computation_values(results['reverse_results'], test['expected_values'], rounding)
-            delta = sum(x['tax_amount_factorized'] for x in results['reverse_results']['taxes_data'] if x['price_include'])
+            # special_mode == 'total_excluded'.
+            if 'total_excluded' not in excluded_special_modes:
+                compare_taxes_computation_values(results['total_excluded_results'], test['expected_values'], rounding)
+                delta = sum(x['tax_amount_factorized'] for x in results['total_excluded_results']['taxes_data'] if x['_original_price_include'])
 
-            self.assertEqual(
-                float_round(results['reverse_results']['total_excluded'] + delta, precision_rounding=rounding),
-                float_round(params['price_unit'], precision_rounding=rounding),
-            )
+                self.assertEqual(
+                    float_round(
+                        (results['total_excluded_results']['total_excluded'] + delta) / params['quantity'],
```

Comment:
ok it was there before, but : why not using assertAlmostEqual instead of assetEqual ?


edit: probably because it's the sum or roundings that messes it up, and we can't just have it right by rounding one digit shorter